setup.py

```diff
@@ -10,9 +10,10 @@
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
+__version__ = "1.4.1"
```

Comment:
Though I think we need to add this `__version__` in `__init.py__` or somewhere so that the following works:
```
import qds_sdk
qds_sdk.__version__
```